game/src/assets/json/characters.json

```diff
@@ -1,30 +1,116 @@
-[
-  {
-    "key": "knight_hero",
-    "name": "Valiant Knight"
-  },
-  {
-    "key": "priest_hero",
-    "name": "Avenging Priest"
-  },
-  {
-    "key": "mage_hero",
-    "name": "Magical Mage"
-  },
-  {
-    "key": "fire_monster",
-    "name": "Fire Monster"
-  },
-  {
-    "key": "ice_monster",
-    "name": "Ice Monster"
-  },
-  {
-    "key": "golem_monster",
-    "name": "Golem"
-  },
-  {
-    "key": "spider_monster",
-    "name": "Spider Monster"
-  }
-]
\ No newline at end of file
+{
+  "characters":[
```

Comment:
It's not necessary for this to be nested under a `characters` key, since the file itself is just for characters.